<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Greedy</class>
 <widget class="QMainWindow" name="Greedy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>950</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>950</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Greedy</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>600</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Daftar Harga Nasi Ayam Di Radius 500m</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="MainTable">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>600</width>
      <height>800</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>42</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>142</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Nama</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Harga (Rp)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Jarak (m)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Harga * Jarak</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="MinItemLabel">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>140</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimal Makan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SolveButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>220</y>
      <width>200</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Cari Dengan Greedy</string>
    </property>
   </widget>
   <widget class="QLabel" name="BudgetLabel">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>180</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Budget</string>
    </property>
   </widget>
   <widget class="QLabel" name="Subtitle">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>80</y>
      <width>400</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Solusi Dengan Algoritma Greedy</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="ResultTable">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>280</y>
      <width>430</width>
      <height>240</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>42</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>142</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Nama</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Harga (Rp)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Jarak (m)</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="TotalPrice">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>550</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="TotalPriceLabel">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>550</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Harga (Rp)</string>
    </property>
   </widget>
   <widget class="QLabel" name="TotalDistance">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>580</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="TotalDistanceLabel">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>580</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Jarak (m)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="BudgetField">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>180</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="MinItemField">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>140</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Time">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>640</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="RunTime">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>640</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Running Time (ms)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>690</y>
      <width>431</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/Trustacean/Downloads/maxresdefault-1875011387.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
